script.service.ltvpl/resources/skins/Default/720p/captureEpgItem.xml

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<window id="1101" type="dialog">
+  <coordinates>
+    <system>1</system>
+    <posx>300</posx>
+    <posy>200</posy>
+    <width>350</width>
+    <height>200</height>
```

Comment:
width / height are not supported by `coordinates`

Reply:
removed